packages/compass-crud/package.json

```diff
@@ -37,7 +37,6 @@
     "bootstrap": "npm run postcompile",
     "prepublishOnly": "npm run compile && compass-scripts check-exports-exist",
     "compile": "npm run webpack -- --mode production",
-    "prewebpack": "rimraf ./dist",
```

Comment:
Moved this to webpack config so that we don't need to keep `rimraf` dependency around